instruments/aiia.yaml

```diff
@@ -0,0 +1,1241 @@
+description: "Het AI Impact Assessment (AIIA) is een hulpmiddel voor het maken van afwegingen bij het inzetten van kunstmatige intelligentie (artificial intelligence, AI) in een project. Het AIIA dient als instrument voor het gesprek en het vastleggen van het denkproces zodat onder andere de verantwoording, kwaliteit en reproduceerbaarheid worden vergroot."
+urn: "urn:nl:aivt:ir:aiia:1.0"
+language: "nl"
+owners:
+  - organization: "Ministerie van Infrastructuur en Waterstaat"
+    name: ""
+    email: ""
+    role: ""
+date: ""
+url: "https://www.rijksoverheid.nl/documenten/rapporten/2022/11/30/ai-impact-assessment-ministerie-van-infrastructuur-en-waterstaat"
+
+
+template: &assessmentcard_template                  # anchor for all fields belonging to answer
+  question: "$QUESTION"
+  answer: "$ANSWER"
+  remarks: "$REMARKS"
+  timestamp: "$TIMESTAMP"
+  authors:
+    - name: "$AUTHOR.NAME"
+      email: "$AUTHOR.EMAIL"
+      role: "$AUTHOR.ROLE"
+
+tasks: # each question can be seen as a separate task
+#Chapter 1: Inleidende vragen
+  - question: "Geef een korte beschrijving van het beoogde AI-systeem (titel, algemene omschrijving, probleemstelling, en het domein)"
+    urn: "urn:nl:aivt:ir:aiia:1.0:1.1"
+    labels:
+      - key: topic
+        value: "Doel van het systeem"
+    type: [ "open_question" ]
+    suggested_roles: ["Opdrachtgever", "Projectleider"] #suggested roles 'Annex: Wie doet Wat' in AIIA are coupled to roles that are described in the Algoritmekader
+    lifecycle: ["Probleemanalyse"]            # one or more lifecycle stages as described in Algoritmekader: https://minbzk.github.io/Algoritmekader/levenscyclus/
+    links: [ {"urn": "urn:nl:aivt:ir:iama:1.0:1.1.1" } ]       # optional: list with links to other tasks / instruments (for example: is this question also asked in the DPIA?)
+    required_status: Yes
+    template: *assessmentcard_template
+  - question: "Waarom is er voor de huidige techniek gekozen? (hierbij is het van belang dat alle afwegingen van robuustheid tot mensenrechten, impact op gebruiker en eindgebruiker, verantwoordingsplicht etc. zijn meegenomen in het antwoord)"
+    urn: "urn:nl:aivt:ir:aiia:1.0:1.2"
+    labels:
+      - key: topic
+        value: "Doel van het systeem"
+    type: [ "open_question" ]
+    suggested_roles: ["Opdrachtgever", "Projectleider"]
+    lifecycle: ["Probleemanalyse" , "Ontwerp"]
+    links: [ {"urn": "urn:nl:aivt:ir:iama:1.0:2B.1.2"}, {"urn": "urn:nl:aivt:ir:iama:1.0:2B.1.3"} ]
+    required_status: Yes
+    template: *assessmentcard_template
+  - question: "Wat is het doel en beoogde resultaat van het AI-systeem?"
+    urn: "urn:nl:aivt:ir:aiia:1.0:1.3"
+    labels:
+      - key: topic
+        value: "Doel van het systeem"
+    type: [ "open_question" ]
+    suggested_roles: ["Opdrachtgever", "Projectleider"]
+    lifecycle: [ "Probleemanalyse" ]
+    links: [ { "urn": "urn:nl:aivt:ir:iama:1.0:1.2.1" }, {"urn": "urn:nl:aivt:ir:iama:1.0:4.4.1"} ]
```

Comment:
Perhaps out of scope for now, but it would also be insightful to include the links to the system_card (and eventually other instruments)

Example: `{ "systemcard_path": ".goal_and_impact", "version": ">0.1a1" }`

Reply:
good one, forgot to add these but I will take a look!